doc/src/tutorial/solvers.rst

```diff
@@ -23,38 +23,48 @@ However, there is an even easier way.  In SymPy, any expression not in an
 = b` if and only if `a - b = 0`, this means that instead of using ``x == y``,
 you can just use ``x - y``.  For example
 
-    >>> solve(Eq(x**2, 1), x)
-    [-1, 1]
-    >>> solve(Eq(x**2 - 1, 0), x)
-    [-1, 1]
-    >>> solve(x**2 - 1, x)
-    [-1, 1]
+    >>> solveset(Eq(x**2, 1), x)
+    {-1, 1}
+    >>> solveset(Eq(x**2 - 1, 0), x)
+    {-1, 1}
+    >>> solveset(x**2 - 1, x)
+    {-1, 1}
 
 This is particularly useful if the equation you wish to solve is already equal
-to 0.  Instead of typing ``solve(Eq(expr, 0), x)``, you can just use
-``solve(expr, x)``.
+to 0. Instead of typing ``solveset(Eq(expr, 0), x)``, you can just use
+``solveset(expr, x)``.
 
 Solving Equations Algebraically
 ===============================
 
-The main function for solving algebraic equations, as we saw above, is
-``solve``.  The syntax is ``solve(equations, variables)``, where, as we saw
-above, ``equations`` may be in the form of ``Eq`` instances or expressions
-that are assumed to be equal to zero.
+The main function for solving algebraic equations as of now is ``solve``.
+The syntax is ``solve(equations, variables)``, ``equations`` may be in the
+form of ``Eq`` instances or expressions that are assumed to be equal to zero.
+We encourage our user to use ``solveset`` as shown above, the syntax for
+``solveset`` is ``solveset(equation, variable=None, domain=S.Complexes)``
 
 .. TODO: This is a mess, because solve() has such a complicated interface.
 
-When solving a single equation, the output of ``solve`` is a list of the
-solutions.
+When solving a single equation, the output of ``solveset`` is a ``FiniteSet`` or
+an ``Interval`` of the solutions.
```

Comment:
Sorry i think i did not made my point clear. Here solution of `sin(x) - 1` is neither a `FiniteSet` nor an `Interval`, but an `infinite set`(an `imageset`). An example of `Interval` returning could be some `trivial` equation like
```
>>> solveset(sin(x)**2 + cos(x)**2 - 1, x, S.Reals)
(-oo, oo)         # an Interval solution
```